### PR TITLE
Restructured media and add page about imagine adapters

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Fixed tickets | none
| License | MIT

#### What's in this PR?

Restructure media and add page about imagine adapters

### CODE REVIEW

File: bundles/media/imagine-adapter.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+This extension uses vips to render the images.
+
+To install it use following commands:
+
+__Mac:__
+
+.. code-block:: bash
+
+    brew install imagemagick
+    pecl install imagick
```
Comment (danrot) [inline bundles/media/imagine-adapter.rst]: Are you sure the above two lines are correct?

File: reference/requirements.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ The following requirements *must* be met to run Sulu:
 * `PHP`_ 7.2 or higher
 * the `intl extension`_ for PHP
 * the `fileinfo` extension for PHP
-* the `gd`_ or `imagick extension`_ for PHP
+* the `gd`_ or `imagick extension`_ or `vips extension`_ for PHP
```
Comment (danrot) [inline reference/requirements.rst]: "gd, imagick_extension or vips_extension"

File: bundles/media/imagine-adapter.rst (hunk 1 above)
Comment (wachterjohannes) [reply bundles/media/imagine-adapter.rst]: yes its correct :)

**Comment by wachterjohannes** [comment]
@danrot @alexander-schranz fixed and ready to review again

File: bundles/media/imagine-adapter.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
```
Comment (danrot) [inline bundles/media/imagine-adapter.rst]: Do you really need to install `imagick` and `imagemagick` if you want to use `vips`? Can't really believe that :thinking:

File: reference/requirements.rst [hunk 4]
```diff
@@ -13,9 +13,9 @@ The following requirements *must* be met to run Sulu:
 * :doc:`Apache <../cookbook/web-server/apache>` or
   :doc:`Nginx <../cookbook/web-server/nginx>` with enabled URL rewriting
 * `PHP`_ 7.2 or higher
-* the `intl extension`_ for PHP
+* the `intl_extension`_ for PHP
 * the `fileinfo` extension for PHP
-* the `gd`_ or `imagick extension`_ for PHP
+* the `gd`_ or `imagick_extension`_ or `vips_extension`_ for PHP
```
Comment (danrot) [inline reference/requirements.rst]: `gd`_, `imagick_extension`_ or `vips_extension`_

File: bundles/media/imagine-adapter.rst (hunk 3 above)
Comment (wachterjohannes) [reply bundles/media/imagine-adapter.rst]: 🙈 overseen that - have changed it

**Pull request merged**